src/sgame/sg_buildable.cpp

```diff
@@ -1075,6 +1075,13 @@ gentity_t *G_GetDeconstructibleBuildable( gentity_t *ent )
 	gentity_t *buildable;
 
 	// Check for revoked building rights.
+	if ( G_admin_permission( ent, ADMF_NO_BUILD ) ) // this is separate from the denybuild check as the
+	{                                               // MN_B_REVOKED phrase doesn't make sense in the context
+	                                                // of the server admin revoking build permissions.
+		trap_SendServerCommand( ent->num(), "print \"You are not permitted to build.\"" );
```

Comment:
nit: `trap_SendServerCommand( ent->num(), "print " QQ( "You are not permitted to build.") );

Reply:
As per the discussion on IRC & in @slipher's review, it would be better to combine the nobuild check with the admin one. The `MN_B_REVOKED` message is not entirely accurate but if this gets approved I'll rebase this PR into two sensible merge-ready commits.

As of https://github.com/Unvanquished/Unvanquished/pull/2356/commits/0575a264686cab9ae08de38ab579b696a985ed78 this code has been removed.